biz.aQute.repository/test/aQute/bnd/repository/p2/provider/P2RepositoryTest.java

```diff
@@ -13,13 +13,15 @@
 
 import aQute.bnd.build.Workspace;
 import aQute.bnd.osgi.Processor;
+import aQute.bnd.service.progress.ProgressPlugin;
 import aQute.lib.io.IO;
 
 public class P2RepositoryTest {
 	File tmp = IO.getFile("generated/tmp");
 
 	@Before
 	public void setUp() {
+		System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "warn");
```

Comment:
This is kind of a big side effect. If you need it in setUp, you better restore the original property in `tearDown`. But I am guessing we don't really need it, it was just for test debugging.